webrender/src/tiling.rs

```diff
@@ -583,13 +584,22 @@ impl AlphaRenderItem {
                         let cache_task_id = picture.render_task_id.expect("no render task!");
                         let cache_task_address = render_tasks.get_task_address(cache_task_id);
                         let textures = BatchTextures::render_target_cache();
-                        let kind = BatchKind::Transformable(
-                            transform_kind,
-                            TransformBatchKind::CacheImage(picture.target_kind()),
+                        let kind = BatchKind::Brush(
+                            BrushBatchKind::Image(picture.target_kind()),
                         );
                         let key = BatchKey::new(kind, blend_mode, textures);
                         let batch = batch_list.get_suitable_batch(key, item_bounding_rect);
-                        batch.push(base_instance.build(0, cache_task_address.0 as i32, 0));
+                        let instance = BrushInstance::new(
```

Comment:
I feel like we'd be better off withing this kind of `new` constructor that's only purpose is to erase the field names. Constructing `BrushInstance {...}` manually here and in other places would read better.

Reply:
Done